scripts/benchmark/checks/simple_value_transfers.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+from typing import (
+    NamedTuple
+)
+
+from eth_typing import (
+    Address
+)
+from evm.chains.base import (
+    Chain
+)
+from evm.rlp.blocks import (
+    BaseBlock,
+)
+
+from .base_benchmark import (
+    BaseBenchmark
+)
+from utils.chain_plumbing import (
+    FUNDED_ADDRESS,
+    FUNDED_ADDRESS_PRIVATE_KEY,
+    get_all_chains,
+    NON_EXISTING_ADDRESS,
+    SECOND_EXISTING_ADDRESS,
+)
+from utils.reporting import (
+    DefaultStat
+)
+from utils.shellart import (
+    bold_yellow
+)
+from utils.tx import (
+    new_transaction
+)
+
+
+class SimpleValueTransferBenchmarkConfig(NamedTuple):
+    to_address: Address
+    greeter_info: str
+
+
+TO_EXISTING_ADDRESS_CONFIG = SimpleValueTransferBenchmarkConfig(
+    to_address=SECOND_EXISTING_ADDRESS,
+    greeter_info='Sending to existing address\n'
+)
+
+
+TO_NON_EXISTING_ADDRESS_CONFIG = SimpleValueTransferBenchmarkConfig(
+    to_address=NON_EXISTING_ADDRESS,
+    greeter_info='Sending to non-existing address\n'
+)
+
+
+class SimpleValueTransferBenchmark(BaseBenchmark):
+
+    def __init__(self, config: SimpleValueTransferBenchmarkConfig) -> None:
+        self.config = config
+
+    @property
+    def name(self) -> str:
+        return 'Simple value transfer'
+
+    def print_result_header(self) -> None:
+        logging.info(bold_yellow(self.config.greeter_info))
+        super().print_result_header()
+
+    def execute(self) -> DefaultStat:
+        number_blocks = 10
+        number_tx = 10
```

Comment:
It looks like you're using a static number of transactions per block.  Can you change this to use `block.gas_limit // 21000` which should be the number of value transfer transactions that can be sent within a single block.